cmd/libs/go2idl/informer-gen/generators/informer.go

```diff
@@ -134,7 +134,7 @@ type $.type|private$Informer struct {
 `
 
 var typeInformerConstructorInternal = `
-func new$.type|public$Informer(client $.clientSetInterface|raw$, resyncPeriod $.timeDuration|raw$) $.cacheSharedIndexInformer|raw$ {
+func new$.type|public$Informer(client $.clientSetInterface|raw$, resyncCheck, resyncPeriod $.timeDuration|raw$) $.cacheSharedIndexInformer|raw$ {
```

Comment:
Why do we need two periods?  Why not derive a resyncCheck from the resyncPeriod?